docs/getting-started/introduction.md

```diff
@@ -1,121 +1,223 @@
-# Getting Started
+# Introduction
 
-> These Docs will help you get up-and-running quickly, guide you through advanced features, and explain the core
-> concepts that make Directus so unique.
+> These Docs will help you get Directus up-and-running quickly, guide you through advanced features, and explain the
+> core concepts that make Directus so unique.
+
+[[toc]]
 
 ## What is Directus?
 
-**Directus is an Open Data Platform built to democratize the database.** Its no-code app is safe and intuitive enough to
-make data accessible to everyone, while still providing the granular configuration and technical tools required by
-power-users and data scientists. For developers, a data connection toolkit provides a dynamic REST+GraphQL API,
-JavaScript SDK, and options for authorization, caching, and more.
+::: tip Directus
+
+Directus ([duh REKT iss](http://audio.pronouncekiwi.com/Salli/Directus)) is Latin for: _laid straight, arranged in
+lines_.\
+The broadest goal of our platform is to present data in a simple, orderly, and intuitive way.
+
+:::
+
+<!--
+@TODO
+getting-started > introduction
+Add 2-4 min Promo Video When Ready
+-->
+
+**Directus is an Open Data Platform built to democratize the database.**
+
+This platform can manage any data, or data model, in any SQL database. Directus uses a technique called
+[Database Introspection](#how-it-works), which identifies your full database structure and creates an abstraction layer,
+then dynamically generates REST and GraphQL API endpoints to manage the data with exhaustive, granular control. These
+APIs are available for devs, but also used to power the intuitive no-code app.
+
+<!--Anything that the APIs can do can also be done in the app and vice versa.-->
```

Comment:
Need to fact-check this statement. As I feel like the API can do a couple extra things the app can't do. @rijkvanzanten